src/apic-extension/azext_apic_extension/aaz/latest/apic/service/_create.py

```diff
@@ -55,21 +55,22 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="The name of the API Center service.",
             required=True,
             fmt=AAZStrArgFormat(
+                pattern="^[a-zA-Z0-9-]{3,90}$",
                 max_length=90,
                 min_length=1,
             ),
         )
 
-        # define Arg Group "Payload"
+        # define Arg Group "Resource"
 
         _args_schema = cls._args_schema
         _args_schema.identity = AAZObjectArg(
             options=["--identity"],
-            arg_group="Payload",
-            help="Managed service identity (system assigned and/or user assigned identities)",
+            arg_group="Resource",
```

Comment:
The API spec defines the parameter name as `resource` instead of `payload`. So the `arg_group` changes here.

Since the spec uses `payload` for other APIs, I'm not sure whether we want to continue using `Resource` as arg group, or change it back to `Payload` to keep it aligned with other commands.

Please let me know your thoughts.